utils/storybook/CanvasProviderDecorator.tsx

```diff
@@ -5,11 +5,21 @@ import {
   createCanvasTheme,
 } from '@workday/canvas-kit-labs-react-core';
 import {object} from '@storybook/addon-knobs';
+import {makeDecorator} from '@storybook/addons';
 
 const label = 'theme';
 
-export default (storyFn: () => React.ReactNode) => (
-  <CanvasProvider theme={createCanvasTheme(object(label, defaultCanvasTheme))}>
-    {storyFn()}
-  </CanvasProvider>
-);
+export default makeDecorator({
+  name: 'canvasProviderDecorator',
+  parameterName: 'canvasProviderDecorator',
+  wrapper: (storyFn, context, {parameters}) => {
+    return (
+      <CanvasProvider
+        disableInputProvider={parameters ? parameters.disableInputProvider : undefined}
```

Comment:
@anicholls I have to check for `parameters` or else every other story fails, the only way to by pass this is if we add the option of `skipIfNoParametersOrOptions` per: https://storybook.js.org/docs/addons/api/ However, if we add this option, canvas provider would never run, this was the only work around I could come up with. I'm open to other options too!